src/domain/registration/permissions.ts

```diff
@@ -47,6 +47,10 @@ export const checkCanUserDoRegistrationAction = ({
   const hasAdminOrg =
     hasAdminOrganization(user) || hasRegistrationAdminOrganization(user);
 
+  if (user?.isSuperuser) {
```

Comment:
Minor optimization and readability improvement suggestion: moving this before the `const hasAdminOrg = ...` line would mean that the `hasAdminOrganization` or `hasRegistrationAdminOrganization` checks would not be done unnecessarily in case of a superuser.

Reply:
I implemented the same improvement to all the other similar functions as well